sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -464,6 +469,14 @@ async Task<bool> Step (MessageId msg_id, StepKind kind, CancellationToken token)
 
 			var res = await SendMonoCommand (msg_id, MonoCommands.StartSingleStepping (kind), token);
 
+			var ret_code = res.Value? ["result"]? ["value"]?.Value<int> ();
+
+			if (ret_code.HasValue && ret_code.Value == 0) {
+				context.CallStack = null;
+				context.IgnoredWrapper = true;
```

Comment:
replace the IgnoredWrapper logic with a call to `await SendCommand (msg_id, "Debugger.resume", new JObject (), token);` here